.ci/smoke-tests.groovy

```diff
@@ -59,6 +59,8 @@ pipeline {
             SSH_KEY = "./id_rsa_terraform"
             TF_VAR_private_key = "./id_rsa_terraform"
             TF_VAR_public_key = "./id_rsa_terraform.pub"
+            TF_VAR_user_name = "${env.BUILD_TAG}"
```

Comment:
**question**: where is the `BUILD_TAG` defined ? 

We need to make sure the `user_name` is meaningful since it is used to tag cloud resources according to the cloud tagging policy.

Reply:
`BUILD_TAG` is provided by jenkins and it's "definition" is:

```
BUILD_TAG - String of "jenkins-${JOB_NAME}-${BUILD_NUMBER}".
```